bedrock/foundation/templates/foundation/trademarks/list.html

```diff
@@ -19,9 +19,15 @@ <h1 class="page-title">{{ _('List of Mozilla Trademarks') }}</h1>
       <th>{{ _('Firefox logo') }}&reg;</th>
     </tr>
     <tr>
+      <td>
+        {{ high_res_img('trademarks/hello-logo-wordmark.png', { 'alt': 'Firefox Hello logo' }) }}
```

Comment:
Yes, I was wondering whether this page is localized. I am guessing it is? If so, this probably needs an l10n label as a heads up to @flodolo